data/json/npcs/random_encounters/free_merchant_caravans.json

```diff
@@ -4,36 +4,69 @@
     "id": "FM_caravan_merchant_random",
     "//": "Random NPC, non persistent, can direct you to the free merchants and sell you things.",
     "name_suffix": "Caravan Merchant",
-    "class": "NC_TRADER",
+    "class": "CARAVAN_MERCHANT",
     "attitude": 0,
     "mission": 3,
     "chat": "TALK_CARAVAN_MERCHANT_1",
-    "faction": "free_merchants",
-    "traits": [ { "trait": "IGNORE_SOUND" }, { "trait": "NO_BASH" }, { "trait": "RETURN_TO_START_POS" } ]
+    "faction": "free_merchants"
   },
   {
     "type": "npc",
     "id": "FM_caravan_guard_A",
     "//": "Random NPC, non persistent, tells you about being a guard and directs you to free merchants.",
     "name_suffix": "Caravan Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "CARAVAN_GUARD",
     "attitude": 0,
     "mission": 8,
     "chat": "TALK_CARAVAN_GUARD_A1",
-    "faction": "free_merchants",
-    "traits": [ { "trait": "IGNORE_SOUND" }, { "trait": "NO_BASH" }, { "trait": "RETURN_TO_START_POS" } ]
+    "faction": "free_merchants"
   },
   {
     "type": "npc",
     "id": "FM_caravan_guard_B",
     "//": "Random NPC, non persistent, doesn't have much to say.",
     "name_suffix": "Caravan Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "CARAVAN_GUARD",
```

Comment:
Since this now uses a new class, you'll also need to define their starting gear. Same with the other new classes.

Reply:
Sorted now, thank you